samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -270,15 +271,15 @@ static int lc3_enable(struct broadcast_sink_stream *sink_stream)
 		return ret;
 	}
 
-	frames_per_sdu = bt_audio_codec_cfg_get_frame_blocks_per_sdu(sink_stream->stream.codec_cfg,
-								     true);
+	frames_per_sdu =
+		bt_audio_codec_cfg_get_frame_blocks_per_sdu(sink_stream->stream.codec_cfg, true);
```

Comment:
Is this just a formatting change?

Reply:
Yes. Auto change done by clang-format -i samples/bluetooth/broadcast_audio_sink/src/main.c. Not needed. I will undo the change

Reply:
If you haven't already, you can usually configure your IDE or clang-format to only format lines changed to avoid things like this :)